### ISSUE TITLE
Community avatars and banners randomly stop displaying

### ISSUE THREAD

**Issue by qoqobolo** [status-im/status-react#20982]
# Bug Report

## Problem 

Sometimes community avatars and banners randomly disappear.
Sometimes it happed right after joining the community, sometimes after some time of using the app (while the user is online), sometimes after re-login.

Sometimes re-login helps the images to start displaying again, but then they disappear anyway in most cases. 
Reproducible for any community, including `Status`, etc

#### Expected behavior
Images do not disappear

#### Actual behavior

All these communities have an avatar and banner

https://github.com/user-attachments/assets/dfd62271-089c-47ed-a3d6-bc199630800c

### Reproduction

No specific steps, just join a community and start using the app sometimes opening the `Communities` screen. 

### Additional Information

- Status version: RC 2.30 
- Operating System: Android, iOS
- Logs: 
[Archive.zip](https://github.com/user-attachments/files/16541119/Archive.zip)

[comment]: # (Please, add logs/notes if necessary)

**Comment by Parveshdhull** [comment]
@qoqobolo is this possible that this issue is related to https://github.com/status-im/status-mobile/issues/20958, and happening due to some network issue etc.

**Comment by pavloburykh** [comment]
Attaching debug logs here

[logs (40).zip](https://github.com/user-attachments/files/16615554/logs.40.zip)
![photo_2024-08-14 18 30 32](https://github.com/user-attachments/assets/7e880229-ef2d-4f4d-9a5e-d03b37d87178)

https://github.com/user-attachments/assets/8e6d7c90-cdbb-4b8d-b213-eb1b7e5bdff4

**Comment by richard-ramos** [comment]
I did not find errors in the log that could indicate a Waku issue. I did see however some `invalid community description outdated clock` which after looking in status-go how this err is generated, could end up ignoring any further change in the community description (including avatars/banners).

**Comment by Parveshdhull** [comment]
@pavloburykh Could you please confirm if this issue is still relevant? It might have been addressed in #21052. (not sure)

**Comment by pavloburykh** [comment]
> @pavloburykh Could you please confirm if this issue is still relevant? It might have been addressed in #21052. (not sure)

@Parveshdhull thank you! We will keep an eye in this and close the issue in case the bug won't come up again. cc @status-im/mobile-qa

**Comment by ilmotta** [comment]
> We will keep an eye in this and close the issue in case the bug won't come up again

FYI @pavloburykh @Parveshdhull I've seen this problem appear many times in iOS in 2.30. @benjthayer also recently reported this problem (https://discord.com/channels/1210237582470807632/1254827811873620068/1282786695309164545)

I don't have exact steps to reproduce, for me it's also kind of random, but sometimes happens more often.

**Comment by pavloburykh** [comment]
> @pavloburykh Could you please confirm if this issue is still relevant? It might have been addressed in #21052. (not sure)

Hey @Parveshdhull! I have just reproduced the issue on my IOS in this [PR](https://github.com/status-im/status-mobile/pull/21242). So I believe the issue is still in develop and thus relevant. 

[logs (46).zip](https://github.com/user-attachments/files/17012459/logs.46.zip)
![photo_2024-09-16 15 03 27](https://github.com/user-attachments/assets/9186ea91-718e-4632-a44e-b201e4114cb7)

https://github.com/user-attachments/assets/f208efbc-067d-44e7-9219-c9b4a8fbebb0

**Comment by VolodLytvynenko** [comment]
I reproduced this issue on Friday too, but I forgot to add it here. I am just confirming that this issue is still reproducible

**Comment by churik** [comment]
[logs (14).zip](https://github.com/user-attachments/files/17211767/logs.14.zip)

1 community - no avatar 

https://github.com/user-attachments/assets/7501c236-6524-41f7-acd2-8446539adff1

**Comment by churik** [comment]
Postponed for the next release due to time constraint and no steps to reproduce. 

cc @ilmotta

**Comment by churik** [comment]
I don't think that we have a capacity for it in 2.32, given the desired release cut date.
Deferring the issue, please be free to move it bak if you think there is a room for it @ilmotta

**Comment by churik** [comment]
It is still the case and it is pretty visible unfortunately, I suggest to prioritize this one for the next release @ilmotta

**Comment by pavloburykh** [comment]
Here’s an example where both the community avatar and a 1-1 chat avatar are missing. Whoever investigates this issue, please let us know whether we should create a separate ticket for the missing 1-1 chat avatar or if it shares the same root cause as the missing community avatar. cc @ilmotta 

[Status-debug-logs (9).zip](https://github.com/user-attachments/files/19108165/Status-debug-logs.9.zip)

https://github.com/user-attachments/assets/a27eb0ff-dc00-48f1-ab06-ce04c9bc4cdd

**Comment by ilmotta** [comment]
I think this issue could be picked up eventually by you @qfrank when you finish your current priorities or whenever is more appropriate for 2.34. Resolving the issue seems to involve a deeper understanding about why the code eventually stops pointing to the correct media server URL, which we suspect could be a the port number issue, although I wouldn't exclude a bug in the CLJS layer.

**Comment by qfrank** [comment]
I doubt I have enough time to investigate this issue this week, unassigned myself now and may pick it up next week
